llvm-spirv/lib/SPIRV/libSPIRV/spirv_internal.hpp

```diff
@@ -101,6 +101,7 @@ enum InternalCapability {
   ICapFPArithmeticFenceINTEL = 6144,
   ICapGlobalVariableDecorationsINTEL = 6146,
   ICapabilityComplexFloatMulDivINTEL = 6414,
+  ICapabilityTensorFloat32RoundingINTEL = 6425,
   ICapabilityTensorFloat32ConversionINTEL = 6425,
```

Comment:
will the two enums having same value cause issues here? Also, may be a TODO here? Thanks

Reply:
oh, it's a cherry-pick mistake